python/util/google.py

```diff
@@ -30,7 +30,7 @@
 <script async src="https://www.googletagmanager.com/gtag/js?id={}"></script>
 <script>
   window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
+  function gtag(){{dataLayer.push(arguments);}}
```

Comment:
do not really understand why this change is needed?

Reply:
this string used as a format string. `{a}` is a format specifier for `a`.